src/support/u128.rs

```diff
@@ -0,0 +1,52 @@
+use crate::support::traits::{CheckedAddInto, CheckedDivInner, CheckedMulInner, CheckedMulInto};
+use primitive_types::U256;
+
+impl CheckedAddInto for u128 {
+    type Output = U256;
+
+    fn checked_add_into(&self, other: &Self) -> Option<Self::Output> {
+        let s = Self::Output::from(*self);
+        let o = Self::Output::from(*other);
+        s.checked_add(o)
```

Comment:
We know that these operations can't fail. I would remove `Option` from the return type. This change would also simplify usage of the methods in another methods.

Reply:
yes, in this particular case, thats correct but the trait is generic where you could have Output as something that could fail.